### PR TITLE
Add Security & Privacy spec section and questionnaire

### PR DESCRIPTION
**Pull request by alcooper91**
Creates a markdown file answering the W3C Security and Privacy Questionnaire as well as translating the Security Considerations sections of the explainer into a relevant "Security and Privacy Considerations" portion of the specification.

***
<a href="https://pr-preview.s3.amazonaws.com/alcooper91/lighting-estimation/pull/39.html" title="Last updated on Nov 19, 2020, 12:41 AM UTC (a3eccf8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/lighting-estimation/39/83f138e...alcooper91:a3eccf8.html" title="Last updated on Nov 19, 2020, 12:41 AM UTC (a3eccf8)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -227,3 +230,22 @@ Event Types {#event-types}
...
+ * Cross Device Communication: A simple broadcast communication method can use device screen or camera LED flashes to broadcast messages read out with lighting estimation on a nearby device.
+
+In addition to these, there are a few vectors unique to lighting estimation to
+consider.
+ * The lighting estimation returned by the WebXR API explicitly describes the real world environment in close proximity to the user.
+ * Reflection cube maps of a high enough resolution approach the same level as camera access.
+
+Lighting estimation must be declared when creating an XR Session as a [=feature descriptor=], which will allow the user agent to notify the user of the potential privacy implications of allowing the lighting estimation API to be used by the website. The user agent is encouraged to NOT provide real-time updates to any portion of the lighting estimation API, especially the reflection cube map. By default, only low spaital frequency and low temporal frequency information should be returned by the WebXR API. Reflection cube maps should be kept low enough texture, unless the user has also consented to camera permissions for a particular origin.
```
Comment (alcooper91) [inline index.bs:249]: The Explainer uses MUST NOT for the real time updates, is there a better place to put this, given that this section should likely be non-normative, per examples?

**Comment by alcooper91** [comment]
@toji Would you mind taking a look?

File: index.bs (hunk 1 above)
Comment (toji) [inline index.bs:249]:
```suggestion
Lighting estimation must be declared when creating an XR Session as a [=feature descriptor=], which will allow the user agent to notify the user of the potential privacy implications of allowing the lighting estimation API to be used by the website. The user agent is encouraged to NOT provide real-time updates to any portion of the lighting estimation API, especially the reflection cube map. By default, only low spaital frequency and low temporal frequency information should be returned by the WebXR API. Reflection cube maps should be kept low resolution, unless the user has also consented to camera permissions for a particular origin.
```

Comment (toji) [reply index.bs:249]: This seems like a fine place for it.

**Review by toji — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (alcooper91) [reply index.bs:249]: Okay, if you're cool with it then that's fine. Since this was non-normative that was my only reservation.

**Pull request merged**